src/libs/actions/Policy.js

```diff
@@ -106,13 +106,30 @@ function getPolicyList() {
     API.GetPolicyList()
         .then((data) => {
             if (data.jsonCode === 200) {
+<<<<<<< HEAD
                 const policyDataToStore = _.reduce(data.policyList, (memo, policy) => ({
                     ...memo,
                     [`${ONYXKEYS.COLLECTION.POLICY}${policy.id}`]: {
                         employeeList: getSimplifiedEmployeeList(policy.value.employeeList),
                         avatarURL: lodashGet(policy, 'value.avatarURL', ''),
                     },
                 }), {});
+=======
+                const policyDataToStore = transformPolicyListToOnyxCollection(data.policySummaryList || []);
+                updateAllPolicies(policyDataToStore);
+            }
+
+            if (shouldCreateNewPolicy) {
```

Comment:
It looks like this variable is missing from the function, so I am thinking there is more code that needs to be CP-ed in a certain order..

Reply:
I think we might be missing this PR: https://github.com/Expensify/App/pull/5706

Reply:
Looks like you're right! https://github.com/Expensify/App/pull/5706 is not present on staging, so we can't CP this to staging without first CP'ing that.